app/models/post.rb

```diff
@@ -1,10 +1,13 @@
+require 'carrierwave/orm/activerecord'
 class Post < ApplicationRecord
-  has_many :comments, dependent: :destroy
+  has_many :comments, dependent: :destroy #post silindiğinde yorumların da silinmesi için
```

Comment:
bu comment e gerek yok. Merak eden dokumantasyona bakabilir.

Reply:
abi ben onu kendim için yazmıştım unutmayim diye, ama sileyim madem

Reply:
git i dogru kullanirsan buna gerek kalmaz :) `git checkout <commit id>` <- bunu yazip istedigin zaman geriye donuk kontrollerini yapabilisin.